code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -108,42 +109,60 @@
 	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2, TECH_ILLEGAL = 8)
 	load_method = MAGAZINE
 	magazine_type = /obj/item/ammo_magazine/c45m
+	allowed_magazines = list(/obj/item/ammo_magazine/c45m)
 
 /obj/item/weapon/gun/projectile/deagle
 	name = "desert eagle"
-	desc = "A robust handgun that uses .50 AE ammo"
+	desc = "A robust handgun that uses .50 AE rounds."
 	icon_state = "deagle"
 	item_state = "deagle"
 	force = 14.0
 	caliber = ".50"
 	load_method = MAGAZINE
+	fire_sound = 'sound/weapons/deagle.ogg'
 	magazine_type = /obj/item/ammo_magazine/a50
-	auto_eject = 1
+	allowed_magazines = list(/obj/item/ammo_magazine/a50)
 
 /obj/item/weapon/gun/projectile/deagle/gold
-	desc = "A gold plated gun folded over a million times by superior martian gunsmiths. Uses .50 AE ammo."
+	desc = "A gold plated gun folded over a million times by superior martian gunsmiths. Uses .50 AE rounds."
 	icon_state = "deagleg"
 	item_state = "deagleg"
 
 /obj/item/weapon/gun/projectile/deagle/camo
-	desc = "A Deagle brand Deagle for operators operating operationally. Uses .50 AE ammo."
+	desc = "A Deagle brand Deagle for operators operating operationally. Uses .50 AE rounds."
 	icon_state = "deaglecamo"
 	item_state = "deagleg"
-	auto_eject_sound = 'sound/weapons/smg_empty_alarm.ogg'
 
+/obj/item/weapon/gun/projectile/fiveseven
+	name = "\improper WT-AP57"
+	desc = "This tacticool pistol makes you feel like you're playing Counter Strike in Space. Uses 5mm rounds."

```

Comment:
I don't understand the issue with pop culture references. SS13 is littered with them--and so is Polaris. Most people seem to appreciate them.